config/vaccines.yml

```diff
@@ -101,13 +101,39 @@ gardasil_9:
 menquadfi:
   brand: MenQuadfi
   type: menacwy
+  discontinued: true
   method: injection
   manufacturer: Sanofi
   nivs_name: MenQuadfi
   dose_volume_ml: 0.5
   snomed_product_code: 39779611000001104
   snomed_product_term: MenQuadfi vaccine solution for injection 0.5ml vials (Sanofi) (product)
 
+menveo:
+  brand: Menveo
+  type: menacwy
+  discontinued: true
+  method: injection
+  manufacturer: GlaxoSmithKline
+  nivs_name: Menveo
+  dose_volume_ml: 0.5
+  snomed_product_code: 17188711000001100
```

Comment:
Not finding this code in the SNOMED CT Browser (but finding `17188711000001105`)